pages/register.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <logo-card>
+    <a-form :form="registerForm" @submit.prevent="register">
+      <a-form-item label="Email" has-feedback>
+        <a-input
+          v-decorator="[
+            'email',
+            {
+              rules: [
+                {
+                  type: 'email',
+                  message: 'Formato de e-mail inválido!'
+                },
+                {
+                  required: true,
+                  message: 'O campo de email é obrigatório!'
+                }
+              ]
+            }
+          ]"
+          placeholder="Email"
+        />
+      </a-form-item>
+
+      <a-form-item label="Senha" has-feedback>
+        <a-input
+          v-decorator="[
+            'password',
+            {
+              rules: [
+                {
+                  required: true,
+                  message: 'Insira uma senha'
+                },
+                {
+                  validator: validateToNextPassword
+                }
+              ]
+            }
+          ]"
+          placeholder="Senha"
+          type="password"
+        />
+      </a-form-item>
+
+      <a-form-item label="Confirme a senha" has-feedback>
+        <a-input
+          v-decorator="[
+            'confirmPassword',
+            {
+              rules: [
+                {
+                  required: true,
+                  message: 'Confirme sua senha'
+                },
+                {
+                  validator: compareToFirstPassword
+                }
+              ]
+            }
+          ]"
+          placeholder="Confirme a senha"
+          type="password"
+        />
+      </a-form-item>
+
+      <a-form-item>
+        <a-button type="primary" html-type="submit" block>
+          Entrar
```

Comment:
```suggestion
          Cadastrar
```
This button should be "Cadastrar" instead of "Entrar"